src/spec_concordius.cpp

```diff
@@ -0,0 +1,85 @@
+/************************************************************************/
+/*  Xania (M)ulti(U)ser(D)ungeon server source code                     */
+/*  (C) 2020 Xania Development Team                                     */
+/*  See the header to file: merc.h for original code copyrights         */
+/************************************************************************/
+
+#include "CHAR_DATA.hpp"
+#include "DescriptorFilter.hpp"
+#include "DescriptorList.hpp"
+#include "comm.hpp"
+#include "merc.h"
+#include "string_utils.hpp"
+#include <fmt/format.h>
+#include <string_view>
+#include <vector>
+
+using namespace fmt::literals;
+namespace {
+
```

Comment:
He has two modes: stand guard & fight. 
In stand guard mode, there's a 1:10 chance (every 4 or 5 seconds) he will do some random emote. If that doens't hit, he may pick one player out and social them. Nothing too fancy yet.